### ISSUE TITLE
Firefox fails to install using the Arch image

### ISSUE THREAD

**Issue by quyykk** [ruanformigoni/flatimage#29]
I'm following the instructions from the README to install Firefox on the Arch image, and at the step where pacman is invoked to install firefox it fails with this:

```
(116/116) checking keys in keyring                              [##################################] 100%
(116/116) checking package integrity                            [##################################] 100%
(116/116) loading package files                                 [##################################] 100%
(116/116) checking for file conflicts                           [##################################] 100%
error: failed to commit transaction (conflicting files)
libvdpau: /usr/lib/libvdpau.so exists in filesystem
libvdpau: /usr/lib/libvdpau.so.1 exists in filesystem
libvdpau: /usr/lib/libvdpau.so.1.0.0 exists in filesystem
Errors occurred, no packages were upgraded.
```

I'm on an Arch host as well if that's relevant. Those files do exist on my host machine. Let me know if you need more information :smile: Thanks!

**Comment by ruanformigoni** [comment]
Hi @quyykk , thanks for the issue report! I'm also on an arch system and was not able to reproduce this issue. Is this version 0.2.4? Try to manually install libvdpau with `./arch.flatimage fim-root fakechroot pacman -S --overwrite "*" libvdpau`, and then install firefox afterwards.

**Comment by quyykk** [comment]
Yes this is with 0.2.4 (sorry for not specifying, I forgot). Using your command I got these warnings but the package installed successfully:

```
warning: warning given when extracting /usr/lib/libvdpau.so (Could not unlink)
warning: warning given when extracting /usr/lib/libvdpau.so.1 (Could not unlink)
warning: warning given when extracting /usr/lib/libvdpau.so.1.0.0 (Could not unlink)
```

And firefox installed and runs without any problems. Nice!

> was not able to reproduce this issue

Just curious, do you have libvdpau installed on the host? Some more information on my system: I'm using Wayland (using sway), and I'm on a laptop with an integrated Intel GPU and a dedicated NVIDIA gpu (but I don't really use the nvidia card).

**Comment by ruanformigoni** [comment]
Ah, I figured it out what happened, libvdpau is bound as read-only to inside the container in case an nvidia gpu is detected, this is why it errors out as "this file exists" and also why it cannot be overwritten. I need to check if nvidia still works ok without this binding to see if I can drop it (and use the one installed though the container instead). Thanks a lot for bringing this to my attention :smile:. I'll send the fix for this in 0.2.5.

**Comment by ruanformigoni** [comment]
Fixed in v0.2.5, package libvdpau and lib32-libvdpau included to avoid host bindings. If the issue persists, feel free to re-open.
